tests/test_redact.py

```diff
@@ -22,12 +23,18 @@ def svs_input_path(data_dir, request) -> Path:
     return data_dir / "input" / request.param
 
 
+@freeze_time("2023-05-12 12:12:53")
+def test_create_redact_dir(tmp_path):
+    output_dir = redact.create_redact_dir(tmp_path)
+    assert output_dir.exists()
```

Comment:
Since this is a unit test, I think it's fine to make stronger assertions about the behavior. I realize that the name is also implicitly tested by the other E2E tests, but it seems reasonable for a unit test to also assert that the name of `output_dir` matches what's expected.

Reply:
added in a0c2bf04704b088fb3ba1cfdeb17ff930b8983bf